test/core/test_em_async_js.c

```diff
@@ -22,17 +22,19 @@ EM_ASYNC_JS(double, foo, (int timeout), {
     () => {
       // Do a simple call back into compiled code synchronously. This is
       // allowed (an async one would not).
-      _inc_result();
+      inc_result();
       // Do the same with ccall.
+      out("XXX");
       ccall("inc_result");
+      out("YYY");
```

Comment:
Temp debugging?

Reply:
Sorry, yes, this is still a draft at this point

Reply:
Oh ok, sorry.

Btw, given the size of the PR, if you can avoid force-pushes over time that would be helpful.